databuilder/databuilder/extractor/delta_lake_metadata_extractor.py

```diff
@@ -425,3 +431,96 @@ def is_array_type(self, delta_type: Any) -> bool:
 
     def is_map_type(self, delta_type: Any) -> bool:
         return isinstance(delta_type, MapType)
+
+    def create_table_watermarks(self, table: ScrapedTableMetadata) -> Union[List[Tuple[Optional[Watermark],
```

Comment:
I think it doesn't really matter which watermark is high and which low so this method could just return Optional[List[Watermark]] which would be more readable and simple. wdyt?